### ISSUE TITLE
Use of hasattr with private variables

### ISSUE THREAD

**Issue by alexalligator** [trix-solutions/moncli#42]
### Background
If a `Board` is instantiated with a list of items then the item ids are stored as a list in a private variable called `__item_ids`.

```python
class Board():

    def __init__(self, **kwargs):
        # ... code hidden

        for key, value in kwargs.items():
           # ... code hidden
            elif key == 'items':
                self.__item_ids = [int(item['id']) for item in value]

            elif key == 'owner':
                self.__owner_id: str = value['id']
```
`Board`'s `get_items` method checks to see whether it already has value for `__item_ids` and if not it fetches them from Monday.

```python
def get_items(self):
        
        field_list = [...]

        if not hasattr(self, '__item_ids'):  # <---- Checking to see if it already has __item_ids

            items = client.get_boards(
                self.__creds.api_key_v2,
                'items.id', 
                ids=[int(self.id)])[0]['items']

            self.__item_ids = [int(item['id']) for item in items]

        items_data = client.get_items(
            self.__creds.api_key_v2, 
            *field_list,
            ids=self.__item_ids, 
            limit=1000)

        return [Item(creds=self.__creds, **item_data) for item_data in items_data] 
```
### Problem
The problem is that `hasattr` with **always** return `False` for variables starting with a double underscore. This is expected behaviour for private variables.

The knock on effect of this is that `get_items` will **always** make the call to Monday, even when this is unnecessary.

### Proposed Solution
Rename `__item_ids` to `_item_ids` or even just `item_ids` so that `hasattr` works as expected

I'm happy to volunteer a PR if you like. 👍

**Comment by Ashatz** [comment]
@alexalligator Thank you for volunteering for a PR!  

Luckily, the fix is simpler than having to do a rename.  I simply identified the name of the private attribute incorrectly.  You should be able to replace '__item_ids' with '_Board__item_ids' to get things working again.  

Please let me know if that does the trick. Thanks!

**Comment by Ashatz** [comment]
@alexalligator, 

This has been resolved for both board.get_items() and group.get_items() and is available with the latest 0.2.5 release.  Thank you for your patience.
